pint/default_en.txt

```diff
@@ -538,23 +538,31 @@ neper = 1 ; logbase: 2.71828182845904523536028747135266249775724709369995; logfa
     cubic_yard = yd ** 3 = cu_yd
 @end
 
+# Note: historically, many of the units in this group were derived from each other (e.g. 1 chain = 10 fathoms = 4 rods = 100 links)
+# but we've followed the NIST's lead here and defined all of the units directly based on "common" units (feet and miles)
+# Source: U.S. Survey Foot: Revised Unit Conversion Factors <https://www.nist.gov/pml/us-surveyfoot/revised-unit-conversion-factors>
```

Comment:
The blog-like post may disappear. The real citation that you could add as proposed at the bottom of https://github.com/hgrecco/pint/issues/1912#issuecomment-1884835524 is:
Warfield, L. , Konijnenburg, J. , Lippa, K. , Lee, G. , Minnich, L. and Williams, J. (2023), Specifications, Tolerances, and Other Technical Requirements for Weighing and Measuring Devices as adopted by the 108th National Conference on Weights and Measures, Handbook (NIST HB), National Institute of Standards and Technology, Gaithersburg, MD, [online], https://doi.org/10.6028/NIST.HB.44-2024, https://tsapps.nist.gov/publication/get_pdf.cfm?pub_id=956954 (Accessed January 11, 2024)

Reply:
In principle the DOI would be enough. Minimal form:
Handbook (NIST HB) - 44-2024, http://dx.doi.org/https://doi.org/10.6028/NIST.HB.44-2024

Reply:
Thank you. I updated both sources accordingly.